package.json

```diff
@@ -43,19 +43,19 @@
     "ui-router-rx": "=0.2.1"
   },
   "peerDependencies": {
-    "@angular/common": "^2.3.1",
-    "@angular/core": "^2.3.1",
+    "@angular/common": "^4.0.0",
+    "@angular/core": "^4.0.0",
     "@angular/router": "^3.3.1"
   },
   "devDependencies": {
-    "@angular/common": "^2.3.1",
-    "@angular/compiler": "^2.3.1",
-    "@angular/compiler-cli": "^2.3.1",
-    "@angular/core": "^2.3.1",
-    "@angular/platform-browser": "^2.3.1",
-    "@angular/platform-browser-dynamic": "^2.3.1",
-    "@angular/platform-server": "^2.3.1",
-    "@angular/router": "^3.3.1",
+    "@angular/common": "^4.0.0",
+    "@angular/compiler": "^4.0.0",
+    "@angular/compiler-cli": "^4.0.0",
+    "@angular/core": "^4.0.0",
+    "@angular/platform-browser": "^4.0.0",
+    "@angular/platform-browser-dynamic": "^4.0.0",
+    "@angular/platform-server": "^4.0.0",
+    "@angular/router": "^4.0.0",
```

Comment:
I have just tested this with angular4 and angular 2 in both cases I got en error
RROR in Error encountered resolving symbol values statically. Calling function 'OpaqueToken', function calls are not supported. Consider replacing the function or lambda with a reference to an exported function, resolving symbol UIROUTER_ROOT_MODULE in /blah/node_modules/ui-router-ng2/lib/uiRouterNgModule.d.ts, resolving symbol _UIROUTER_INSTANCE_PROVIDERS